liquibase/changes/feat_565.sql

```diff
@@ -0,0 +1,17 @@
+ALTER TYPE datatype ADD VALUE 'gbfs';
+
+CREATE TABLE GBFS_Feed(
```

Comment:
[picky]: Just for consistency name the table `GBFSFeed` like `GTFSFeed`. The Model name is generated as `GbfsFeed` vs `Gbfsfeed`. I agree that this proposal is a better naming convention; However is different that the current one in addition the Model generated `GBFSFeed` has the same pattern than the API model that would have been generated(not currently as it's not defined in the OpenApiSchema)

Reply:
Agreed! I updated the naming